CONTRIBUTING.md

```diff
@@ -79,8 +79,8 @@ in order to install local versions of husky, lint-staged, and eslint.
 It is recommended that you also install ts-node and typescript globally, e.g.
 
 ```shell
-npm install -g typescript
-npm install -g ts-node
+npm install -g typescript@4.6.4
+npm install -g ts-node@10.8.0
```

Comment:
One worry I have here is these getting out of date. It's going to be extremely likely that somebody will forget to bump this documentation in the future. This seems like the same problem, but in reverse. People will install this old version and then it won't work with whatever the new version needs.

I think this is more likely of a problem than the one you ran into, so I'd prefer not to change the versions.

(Happy to discuss more too, or maybe there's some solution to keep this doc up to date or a command that installs the package.json version globally that I don't know about?)

Reply:
Ah, true, that is something I should have considered. I looked around to see if I could find a better solution, but I couldn't find anything cleaner than simply listing the most recent version. Putting it that way, I definitely agree that leaving it out is the better option for now.